arduino-ide-extension/src/node/config-service-impl.ts

```diff
@@ -217,13 +221,20 @@ export class ConfigServiceImpl
 
   private async getFallbackCliConfig(): Promise<DefaultCliConfig> {
     const cliPath = this.daemon.getExecPath();
-    const rawJson = await spawnCommand(cliPath, [
+    const rawJson = await spawnCommand(cliPath, ['config', 'dump', '--json']);
+    const config = JSON.parse(rawJson);
+
+    // Since CLI 1.0, the command `config dump` only returns user-modified values and not default ones.
+    // directories.user and directories.data are required by IDE2 so we get the default value explicitly.
+    const directoriesRaw = await spawnCommand(cliPath, [
```

Comment:
The CLI executions do not depend on each other: they can spawn parallel with `Promise.all()`